head/src/main/resources/views/fragments/campaign-summary.html

```diff
@@ -7,25 +7,25 @@
         <div class="float-left campaign-summary-top-first">
             <div class="campaign-summary-title">
                 <span class="float-left color"></span>
-                <h3 class="float-left pb-4" th:text="${campaign.name}">No Campaign</h3>
+                <h3 class="float-left pb-3" th:text="${campaign.name}">No Campaign</h3>
             </div>
-            <div class="float-left button-wrapper" th:replace="fragments/button :: button (${campaign.result})"
+            <div class="float-left button-wrapper" th:replace="~{fragments/button :: button (${campaign.result})}"
                  th:classappend="${campaign.result}">
                 <span>Finished</span>
             </div>
         </div>
         <span class="float-left vertical-divider"></span>
         <div class="float-left campaign-summary-top-second">
             <div class="float-left campaign-summary-top-second-items campaign-summary-top-second-items-first">
-                <img class="float-left img" style="margin-right: 8px;" th:src="${timeImage}" src="${timeImage}"
-                     alt="Time icon"/>
+                <img class="float-left img" style="margin-right: 8px; padding-top: 2px;" th:src="${timeImage}"
```

Comment:
Is there a way to use a different attribute than alt? Because the users see its content when hovering the image as HTML.

Reply:
Although alt is the standard attribute for img tags. 
But can the users ever hover over this image as HTML? I don't think so....
On render, I send the ID of the report task and on download, i send the pdf as byte array. I think any hover would come from the frontend. Don't you think so?

Reply:
Let's keep it for the moment, since the report cannot be downloaded as HTML but only as PDF.